ShdrPlygrnd/materials/paper.mtlx

```diff
@@ -47,14 +47,14 @@
   </normalmap>
 
   <open_pbr_surface name="mtlxopen_pbr_surface" type="surfaceshader">
-    <!-- FIXME: For paper like maeerial using thin wall is ideal ! -->
     <input name="base_color" type="color3" nodename="BaseColor" />
     <input name="specular_color" type="color3" value="0.42767, 0.42767, 0.42767" />
     <input name="specular_roughness" type="float" value="1" />
     <input name="subsurface_weight" type="float" nodename="SSSweight" />
     <input name="subsurface_color" type="color3" nodename="mtlxcolorcorrect1" />
     <input name="subsurface_radius" type="float" value="0.05" />
     <input name="geometry_normal" type="vector3" nodename="mtlxnormalmap1" />
+    <input name="geometry_thin_walled" type="boolean" value="true" />
```

Comment:
Happy to see your trust :) Have you noticed any meaningful differences with this ?

Reply:
Agreed on all your observations, to me the thin walled makes much more sense, and yes because of the Openpbr "2 in 1" material :)
Thanks for sharing renders with different combinations